conversion.c

```diff
@@ -108,31 +104,33 @@ int Call (int num, char* bin, char* hex) {
 	printf ("BIN : %s\n", bin);
 	printf ("HEX : %s \n", hex);
 	printf ("\n");
```

Comment:
Can print all statements in a single printf
Example: printf("INPUT : %d\nBIN : %s\nHEX : %s\n",num, bin, hex);